<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeechToText</class>
 <widget class="QMainWindow" name="SpeechToText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpeechToText</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
    background-color: #fdfaf5;
    font-family: 'Helvetica', sans-serif;
    }

    QPushButton {
    background-color: #007acc;
    color: white;
    border-radius: 8px;
    padding: 6px 14px;
    font-weight: bold;
    font-size: 13px;
    }

    QPushButton:hover {
    background-color: #005f99;
    }

    QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 6px;
    padding: 4px 8px;
    }

    QTextEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 8px;
    padding: 8px;
    color: #333333;
    font-size: 15px;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnTranscribe">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Transcribe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFilePath">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOutput">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>491</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecord">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPause">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveTranscript">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Transcript</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1512</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
